nav2_mppi_controller/include/nav2_mppi_controller/tools/noise_generator.hpp

```diff
@@ -84,6 +79,10 @@ class NoiseGenerator
    */
   void reset(mppi::models::OptimizerSettings & settings, bool is_holonomic);
 
+  inline static std::normal_distribution<float> ndistribution_vx;
```

Comment:
Why inline statics? There should only be 1 instance of the noise generator

Reply:
Oops, by mistake. I'll correct it.